### DISCUSSION TITLE
大佬牛x，表示支持，希望不停更新完善！

### DISCUSSION THREAD

**Discussion by wfniub** [RipplePiam/MobaXterm-Chinese-Simplified#9] [General]
大佬牛x，表示支持，希望不停更新完善！这个软件确实好用！

**Comment by wfniub** [comment]
不过还是不够完善，可以参考52pojie里面的那个汉化比较完善，只是不是最新版本的！

**Comment by RipplePiam** [comment]
除了对话框提示信息，其他几乎汉化完成了。
目前能看到的未汉化内容，一部分是诸如 TextEditor 这样的插件，一部分是右键菜单，还有部分组件；
前两者是二进制内容，此项目只做汉化，不做反编译，也不做破解，故不会去汉化。
后者是软件本身的Bug，内容其实已经汉化。
如有其他具体问题请详细描述。
感谢你的认可。
